packages/common/client-utils/src/storageUtils.ts

```diff
@@ -0,0 +1,36 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IPartialSnapshotWithContents } from "@fluidframework/driver-definitions";
+import { ISnapshotTree } from "@fluidframework/protocol-definitions";
+
+/**
+ * Utility API to check if the type of snapshot contents is `IPartialSnapshotWithContents`.
+ * @alpha
+ * @param obj - obj whose type needs to be identified.
+ */
+export function instanceOfIPartialSnapshotWithContents(
+	// eslint-disable-next-line @rushstack/no-new-null
+	obj: ISnapshotTree | IPartialSnapshotWithContents | null | undefined,
+): obj is IPartialSnapshotWithContents {
+	return (
+		obj !== null &&
+		obj !== undefined &&
+		"couldBePartialSnapshot" in obj &&
+		obj.couldBePartialSnapshot === true
+	);
+}
+
+/**
+ * Utility API to check if the type of snapshot is `ISnapshotTree`.
+ * @alpha
+ * @param obj - obj whose type needs to be identified.
+ */
+export function instanceOfISnapshotTree(
```

Comment:
Do we need to have that function? I do not se any usage.
I can see why it might be interesting for debugging (asserts), but it feels like there should be no usage in production code, so maybe not add it?

Reply:
Removed